win32ss/user/rtl/text.c

```diff
@@ -1263,20 +1263,8 @@ INT WINAPI DrawTextExWorker( HDC hdc,
 	if (flags & DT_SINGLELINE)
 	{
 #ifdef __REACTOS__
-        if (flags & DT_VCENTER)
-        {
-            if (flags & DT_CALCRECT)
-            {
-                if (rect->bottom - rect->top < size.cy / 2)
-                    y = rect->top + (invert_y ? size.cy : -size.cy) / 2;
-            }
-            else
-            {
-                y = rect->top + (rect->bottom - rect->top + (invert_y ? size.cy : -size.cy)) / 2;
-            }
-        }
-        else if (flags & DT_BOTTOM)
-            y = rect->bottom + (invert_y ? 0 : -size.cy);
+	    if (flags & DT_VCENTER) y = (rect->top + rect->bottom - size.cy - 1) / 2;
+	    else if (flags & DT_BOTTOM) y = rect->bottom - size.cy;
```

Comment:
Does it make the test mentioned in your past commit 11b7619a still working?

Reply:
Oh, I've forgotten it. I will test.